### ISSUE TITLE
Fix alignment of tooltips in "Edit course" and "New course" forms on mobile screens

### ISSUE THREAD

**Issue by a22moamy** [HGustavs/LenaSYS#15621]
Currently, the tooltips/dialogs in "Edit course" and "New course" look great on wider screens. However, they overlap with the inputs on mobile devices / smaller screens. 
The current appearance:
![Skärmavbild 2024-05-13 kl  10 42 17](https://github.com/HGustavs/LenaSYS/assets/129263915/7c558eb0-ef00-4998-983c-c18d895343c3)

Please fix this with in any way seen fitting, as long as the solution uses media queries. You can decide whether your solution is  pushing the tooltips more to the left or making them appear under the inputs like this: 
![Skärmavbild 2024-05-13 kl  10 46 41](https://github.com/HGustavs/LenaSYS/assets/129263915/c76fefa1-b933-4d99-82b9-2c6b7c912602)
(Screenshot taken from Datorgrafik when editing a new Link)

Also, make sure the CSS is not broken for wider screens **and for other forms**.

## For tester (and developer)
Test this issue by adjusting the screen to many different sizes and checking the appearance of the tooltips. Also make sure no tooltips from **other forms** have been broken by the fix.

**Comment by a22chrfa** [comment]
@a22moamy I've been looking into this and I've started tinkering around.

It seems many (if not all) validation messages (black boxes) on the site has been done with specific pixel adjustments made to them. Rather than using something like flexbox with automatic aligments everywhere. Which generally isn't a good solution.

I used flexbox and removed the specific aligments for the "edit course"-box, which made it look neat. But as expected, this made the validation texts in other parts of the site misaligned and messed up since they now have the flexbox settings, but also still have their specific pixel adjustments.

I could fix the specific fields you mentioned by modifying their pixel values, but I don't consider this a longterm solution, and not good coding. I'd rather we set up the flexbox for all validation messages and I'll try to remove as many specific pixel adjustments I can find. If we find any in the future, these can then easily be fixed by removing their specific alignments (mostly found in inline-styling).
